docs/podman-stats.1.md

```diff
@@ -33,44 +33,49 @@ Disable streaming stats and only pull the first result, default setting is false
 
 Pretty-print images using a Go template
 
+Valid placeholders for the Go template are listed below:
```

Comment:
Could you add a template example or two below?  Also the example has a formt=json example, but no mention of jsno anywhere else in the man page.  Can you add that too please?

Reply:
fixed